### ISSUE TITLE
[BUG] Cannot mix controls and dragging

### ISSUE THREAD

**Issue by angry-dan** [framer/motion#413]
**2. Describe the bug**

I can no longer drag after using controls. This might be intentional, but I can't see anything in the docs about the relationship between controls and dragging. I came across the undocumented _dragTransitionControls, but I don't know how to use it.

**3. IMPORTANT: Provide a CodeSandbox reproduction of the bug**

https://codepen.io/angry-dan/pen/RwNoJMM

**4. Steps to reproduce**

Visit the Codepen above, note that you can drag the red box but only once. The onDragComplete starts an animation and after that point dragging is no longer possible.

**5. Expected behavior**

You drag the element, when you release it moves to the defined position, then you can continue to drag it.

Obviously in the case of this simple example I can use a drag constraint. What I'm trying to do though is to complete a swiping transition that the user makes. Similar to the example here: https://codesandbox.io/s/framer-motion-image-gallery-pqvx3, I want a user to swipe an element and then if that element is swiped far/fast enough have it transition to its final position. I tried with dragTransition and modifyTarget but they just created really weird behaviours.

**Comment by angry-dan** [comment]
So I just learnt something new about this bug, in that there's an incompatibility between drag and setting offset transforms. I'm not clear on exactly how or why, but the simplest way I can replicate it is here: https://codepen.io/angry-dan/pen/NWPwPQZ. Simply setting `y` with initial prevents the first drag operation from working. 

I hadn't noticed it at first, but this example also suffers from the same bug: https://codesandbox.io/s/framer-motion-bottom-sheet-fixed-m2vls. I'm pretty sure that my first codepen link above and this second example are closely related.

**Comment by angry-dan** [comment]
I did a bit more research into this, and found the root cause. I've opened a new issue here: https://github.com/framer/motion/issues/424 so closing this one.
